i18n/en.ts

```diff
@@ -384,6 +384,58 @@ Saturdays & Holidays: 09.00 am to 05.00 pm`,
     },
     sections: { clubs: { title: 'CLUBS', more: 'Explore all clubs' } },
   },
+  Unauthorised: {
+    title: 'Unauthorised',
+    description: 'You are not authorised to view this page.',
+  },
+  WorkInProgress: {
+    title: 'Work In Progress',
+    description:
+      'This page is currently being worked on. Please visit another time',
+  },
+
+  infrastructure: {
```

Comment:
Since this is inside `/institute`, the translation object should be within `Institute` as well. And any solo page of its own should have a capitalised key named based on the page, so this would be `CampusAndInfrastructure`